lb_logger.py

```diff
@@ -0,0 +1,9 @@
+import logging
+"""
+Instantiates a logger object with level set to INFO
+
+This will be used in all other files as the default logger object
+"""
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
don't need this if we go with default level UNLESS client instantiation changes it

Reply:
commented on this inside the client.py file conversation